### PR TITLE
[General] Check epic status and show warning messages

### PR DESCRIPTION
**Pull request by flavioislima**
This PR checks Epic Status API and if the Status is Outage show some warning messages to the user on Install, Import and Update games. And others on the console for Sync Saves, etc.

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: electron/main.ts [hunk 1]
```diff
@@ -788,6 +791,18 @@ ipcMain.handle('install', async (event, params) => {
...
+    dialog.showErrorBox(
+      i18next.t('box.warning.title', 'Warning'),
+      i18next.t(
+        'box.warning.epic.install',
+        'Epic Servers are having major outage right now, the game cannot be installed!'
+      )
+    )
+    return { status: 'error' }
+  }
+
```
Comment (Nocccer) [inline electron/main.ts:805]: You adding alot of these with just different i18n keys. Maybe make it a function and reuse it. Would make the code more adjustable and readable

**Review by Nocccer — CHANGES_REQUESTED** [review]
Beside the comment i left lgtm. If you don't like the note i left, hit me up so i can approve it or just merge it.

File: electron/main.ts (hunk 1 above)
Comment (flavioislima) [reply electron/main.ts:805]: Yes, might be but i18next has some issues with encapsulated or keys that are under some kind of check.
Like: `varTrue ? 'key.one' : 'key.two'`
Once we run the command to generate the new keys, it just ignore this and none are created.

Comment (flavioislima) [reply electron/main.ts:805]: the same with:
`box.warning.epic.{action'}`
it doesn't created the `actions` keys

**Pull request merged**